healthtools/scrapers/health_facilities.py

```diff
@@ -73,63 +48,49 @@ def get_token(self):
-            # Push the data to s3
-            self.archive_data(payload)
+            res = requests.get(SEARCH_URL, headers=headers)
+            data = res.json()
+            results = data['results']
 
-            # Push the delete payload to s3
-            delete_file = StringIO(delete_payload)
-            self.s3.upload_fileobj(
-                delete_file, "cfa-healthtools-ke",
-                self.delete_file)
+            print "{{{0}}} - Scraper completed. {1} documents retrieved." \
+            "".format(datetime.now().strftime('%Y-%m-%d %H:%M:%S'),len(data['results']))
+            self.push_to_elasticsearch(results[0:2])
+            self.archive_data(json.dumps(results))
             print "{{{0}}} - Completed Scraper.".format(datetime.now().strftime('%Y-%m-%d %H:%M:%S'))
 
         except Exception, err:
             print "ERROR IN - index_for_search() - %s" % (err)
 
-    def index_for_cloudsearch(self, record):
-        return health_facilities_template  % (
-            record['code'],
-            record['name'].replace("\"", "'"),
-            record['facility_type_name'],
-            record['approved'],
-            record['sub_county_name'],
-            record['service_names'],
-            record['county_name'],
-            record['open_public_holidays'],
-            record['keph_level_name'],
-            record['open_whole_day'],
-            record['owner_name'],
-            record['constituency_name'],
-            record['regulatory_body_name'],
-            record['operation_status_name'],
-            record['open_late_night'],
-            record['open_weekends'],
-            record['ward_name'].decode("string_escape").replace('\\', ''),
-            )
-
-    def delete_payload(self, record):
-        return delete_template %(record['code'])
+    def format_doc(self, entry):
+        return {
+            "id":entry['code'],
+            "name":entry['name'].replace("\"", "'"),
+            "facility_type_name":entry['facility_type_name'],
+            "approved":entry['approved'],
+            "sub_county_name":entry['sub_county_name'],
+            "service_names":entry['service_names'],
+            "county_name":entry['county_name'],
+            "open_public_holidays":entry['open_public_holidays'],
+            "keph_level_name":entry['keph_level_name'],
+            "open_whole_day":entry['open_whole_day'],
+            "owner_name":entry['owner_name'],
+            "constituency_name":entry['constituency_name'],
+            "regulatory_body_name":entry['regulatory_body_name'],
+            "operation_status_name":entry['operation_status_name'],
+            "open_late_night":entry['open_late_night'],
+            "open_weekends":entry['open_weekends'],
+            "ward_name":entry['ward_name'].decode("string_escape").replace('\\', '')
+            }
```

Comment:
In ElasticSearch, the above might not be necessary specifically for health facilities that seems well formatted already from the API.